src/app/related-search/related-search.component.html

```diff
@@ -1,7 +1,17 @@
 <div *ngIf="results?.length > 0" class="card">
     <h3>Searches related to {{this.keyword}}</h3>
     <div *ngFor="let result of results" class="related">
-      <a [routerLink]="resultsearch" [queryParams]="{query: result.label}" class="rellink">{{result.label}}</a>
+      <a [routerLink]="resultsearch" [queryParams]="{query: result.label,
+                                                      verify: false,
+                                                      nav: 'filetype,protocol,hosts,authors,collections,namespace,topics,date',
+                                                      start: 0,
+                                                      indexof: 'off',
+                                                      meanCount: '5',
+                                                      resource: 'global',
+                                                      prefermaskfilter: '',
+                                                      rows: 10,
```

Comment:
Also, can you please elaborate? What would you want me to define in the .ts file exactly? The data types, or do you think the data should be pre-defined too? If you think the data should be predefined in the .ts file, please note that the query changes with every link, and hence that won't be possible, unless I write a special function on (click). I thought I would avoid all the extra lines of code, by writing it in the html file itself. So please elaborate on what exactly you want changed. :)